Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -104,15 +104,20 @@ script:
         Example: 8.8.8.8,1.1.1.1.
       isArray: true
       name: ip_list
```

Comment:
Is there a difference between ip_list and public_ip_list?

Reply:
ip_list is for private IP addresses and public_ip_list is for public IP addresses (also know as "last origin IP")